.changeset/stupid-turtles-invite.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
```

Comment:
Let's just keep `@finos/legend-graph` and `@finos/legend-studio` here. 
We can add an empty changeset with no message that has 
`@finos/legend-manual-tests`

Reply:
Changed [here](https://github.com/finos/legend-studio/pull/1010/commits/726df0dc39fe0206abcaf4a5f5c4eb4aeac086b4)